src/components/VolumeViewerWrapper.jsx

```diff
@@ -0,0 +1,32 @@
+import { Container } from "react-bootstrap";
+import { VolumeViewer } from "react-volume-viewer";
+import {
+  ALL_COLOR_MAPS,
+  MODEL_CONSTANTS,
+  MODEL_DATA,
+} from "../constants/constants";
+
+export default function VolumeViewerWrapper(props) {
+  const { colorMap, selection, controlsVisible } = props;
+
+  const model = {
+    ...MODEL_CONSTANTS,
+    path: `./assets/models/${selection.season.value}-${selection.tide.value}-${selection.measurement.value}.png`,
+    range:
+      MODEL_DATA[
+        `${selection.season.value}-${selection.tide.value}-${selection.measurement.value}`
```

Comment:
would it be worth pulling this concatenation out above this `model` declaration as it's used also in path?

Reply:
Yeah that's an easy fix, called it `fileName`